cli/cli_test.go

```diff
@@ -35,6 +35,16 @@ type cliTest struct {
 	*server.TestServer
 }
 
+// Stop the server, or wait until stopping is done
+// if it was already initiated by the caller.
+func (c *cliTest) cleanup(wasStopped *bool) {
+	if *wasStopped {
```

Comment:
why not just `<-c.Stopper().IsStopped()` after the quit command? I kindda like that we're testing it there too. Does calling `c.Stop()` buy us anything?